### ISSUE TITLE
Insight - "No Data Available" 

### ISSUE THREAD

**Issue by itheiss** [opnsense/core#2109]
Hi,

am on OPNsense OPNsense 18.1.r2 and since OPNsense 18.1.r1 I'am getting "No Data Available" from Insight. The issue is the same as #1696 but "Reset Netflow data" and "service flowd_aggregate restart" don't help. I also rebooted but no difference.

If Iook at /var/log/flowd.log the is only binary output inside. Is this normal?

````
root@opnsense:~ # head -10 /var/log/flowd.log
??d??5%Ln$??hZ`??$??hZ`?$֭?$֭?  `??Z`??S??d[QhQ???	?&$?.?Z`??	$֭?$֭? `??Z`?????b5Y$?.?Z`??	$֭?$֭? `??Z`????%L5N$?.?Z`??	$֭?$֭? `??Z`?????\58$?.?Z`??	$֭?$֭? `??Z`??S5????d??P???$?.?Z`??	$??$??h  `??Z`??S??d5??????P?$?.?Z`??	$??$??h `??Z`??S??d5??????[$L$?.?Z`????$??h $?HX$??P  `??Z`??S??d??d[?????Q$?.?Z`??P$?HX$??P  `??Z`??S??d??d[?????u$?.?Z`???$?HX$??8  `??Z`??S??d??d[???8,$?.?Z`??	$?HX$??8  `??Z`??SQhQ??d[????	h$?.?Z`??	$??$??  `??Z`??S??d[>?d????????$?.?Z`??	$?c?$? `??Z`??S>?d???d[?????amg$?.?Z`??	$?c?$?0  `~$?.?Z`??dQ?????$?0$?0  `??Z`??S??d[?O????Z`??SQ?????d??ㆫ
                                          ,$?.?Z`??	$֭?$?' `??Z`??S?O???d[_$?.?Z`??	$֭?$?'  `??Z`??S5????d??P??h$?.?Z`??	$??h$??h  `??Z`??S5????d??P??4$?.?Z`??	$??h$??h  `??Z`??S??d??????????$?.?Z`??	$??P$??P `??Z`??S??????d??????$?.?Z`??	$??P$??P  `??Z`??U????)???JP?H7$ק?Z`??	$?Ǹ$֖x `??Z`??U???h?????7N?4$ק?Z`??	$?_?$?_? `??Z`??U???@駼???l($ק?Z`??	$?oh$?oh `??Z`??U???h?[?????x$ק?Z`??	 `??Z`??U?????????	???$ק?Z`??	$֎?$֎? `??Z`????J?58$ק?Z`??	$֭?$֭? `??Z`??U???QhQ??z??&$ק?Z`??	$֭?$֭? `??Z`????h!5N$ק?Z`??	$֭?$֭? `??Z`????s05J$ק?Z`??	$֭?$֭? `??Z`?????m5Y$ק?Z`??	$֭?$֭? `??Z`??U???>?d????????$ק?Z`??	$?c?$? `??Z`??U???Q?????8Y?J$ק?Z`??	$?0$?0 `??Z`??U????O???b??
                        ,$ק?Z`??	$֭?$?' `??Z`??U?????????	???$ק?Z`??	$??P$??P `??Z`??U???h?[?????($ק?Z`??	$??8$??8 `??Z`??U???>?d?????s$ק?Z`??	$??@$?6? `??Z`??U???QhQ??z??h$ק?Z`??	$??$?? `??Z`??U???>?d??????4$ק?Z`??	$?0$?0 `??Z`??U???Q?????8Y?4$ק?Z`??	$?0$?0 `??Z`??U???@駼???l($ק?Z`??	$?0$?0 `??Z`??U????O???b??4$ק?Z`??	$?'$?' `??Z`??U???>?d?????4$ק?Z`??	$?6?$?6? `??Z`??U?????????	???$ק?Z`??	$?Q?$?Q? `??Z`??U???h?[?????($ק?Z`??	$?ih$?ih `??Z`?D~????dI??P?h$???Z`?	$?.?$?.?  `??Z`?D~h?[??da????S($???Z`?	$??8$??8  `??Z`?D~??dah?[???S?($???Z`?	$??8$??8 `??Z`?D~????dI??P?h$???Z`?	$?.?$?2?  `??Z`?D~??d[>?d?????s$???Z`?	$??@$?6? `??Z`?D~>?d???d[????
           ?$???Z`?	$??@$?6?  `??Z`?D~????dI??P?h$???Z`?	$?2?$?6?  `??Z`?D~??d[QhQ???	?h$???Z`?	$??$?? `??Z`?D~????dI??P?h$???Z`?$?>p$?J(  `??Z`?D~??d[>?d??????4$???Z`?	$?0$?0 `??Z`?D~@駼??d[??l??4$???Z`?	$?0$?0  `??Z`?D~??dQ????????4$???Z`?	$?0$?0 `??Z`?D~Q?????d??ㆫ4$???Z`?	$?0$?0  `??Z`?D~??d[@駼????l($???Z`?	$?0$?0 `??Z`?D~??d[?O????4$???Z`?	$?'$?' `??Z`?D~??d^????????l?$???Z`?	$?2?$?2? `??Z`?D~??d[>?d?????4$???Z`?	$?6?$?6? `??Z`?D~????dI??P?h$???Z`?	$?]?$ׄ?  `??Z`?D~??d??????????$???Z`?	$?Q?$?Q? `??Z`?D~??????d?????T$???Z`?	$?Q?$?Q?  `??Z`?D~h?[??da????S($???Z`?	$?ih$?ih  `??Z`?D~??dah?[???S?($???Z`?	$?ih$?ih `??Z`?D~??dQ????????J$???~$???Z`?$???$???$???$???  
```

Any thoughts?

Regards
Ingo

**Comment by fichtner** [comment]
Fixed in 18.1. Workaround here: https://forum.opnsense.org/index.php?topic=6879.msg30037#msg30037


Cheers,
Franco

**Comment by itheiss** [comment]
thx
